cves/kernel/CVE-2014-0206.yml

```diff
@@ -55,7 +55,8 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  By lying about the nature of the data it was possible for stack memory to be returned to a low level user
```

Comment:
Elaborate on this. What kind of "Lie" is needed to mess with the stack memory. What data would be returned. What implications does this vulnerability have ? Remember not to include too much jargon related to the linux kernel in general.